.papr.sh

```diff
@@ -126,4 +127,5 @@ if [ $integrationtest -eq 1 ]; then
     make TAGS="${TAGS}" test-binaries
     make varlink_generate GOPATH=/go
     make ginkgo GOPATH=/go $INTEGRATION_TEST_ENVS
+    make ginkgo-remote GOPATH=/go $INTEGRATION_TEST_ENVS
```

Comment:
I'm assuming this is special for `.papr` and not needed in `integration_test.sh`?